client/src/js/components/map.jsx

```diff
@@ -39,7 +39,17 @@ var Map = React.createClass({
       }
     ).addTo(map);
 
-    L.geoJson(items).addTo(map);
+    var marker = new 
+    L.Icon({
+      iconUrl: "images/marker-icon.png"
+    });
+
```

Comment:
Both single and double quotes are valid in JS, but convention is typically to use single-quotes and reserve double quotes for html attributes in your JS.